CI/Dockerfile_1_external_deps

```diff
@@ -1,6 +1,6 @@
 ARG UBUNTU_VERSION=18.04ARG
 ARG OWNER=svalinn
-FROM ghcr.io/${OWNER}/dagmc-ci-ubuntu-${UBUNTU_VERSION}
+FROM ghcr.io/${OWNER}/dagmc-ci-ubuntu-${UBUNTU_VERSION}:ci_testing
```

Comment:
Should we pass this in as an argument so that these scripts can be used in other contexts?